### ISSUE TITLE
feat(patch): Working Fullscreen quick actions on tablets

### ISSUE THREAD

**Issue by zzahkaboom24** [revanced/revanced-patches#2170]
### Application

YouTube ReVanced

### Issue

Not working quick actions in fullscreen mode when watching a YouTube video. First Screenshot is on tablet, second screenshot is on mobile for comparison.

![Screenshot_20230513_201356](https://github.com/revanced/revanced-patches/assets/42496614/de6456b0-4236-49d9-8508-ba66371602ef)

![Screenshot_20230513_201938](https://github.com/revanced/revanced-patches/assets/42496614/5dba3d07-9253-4fdb-a9c3-c4c354b0553f)

Note that quick actions in fullscreen do not work in the normal YouTube client on tablets, whereas they do work on mobile. I hope this is achievable for tablets as well somehow.

### Patch

Basically the quick actions in fullscreen patch, but working for tablets as well.

### Motivation

It adds quick actions when watching videos in fullscreen basically improving the "productivity" when in fullscreen. I also think tablet users would appreciate when this feature actually works for them as well and not keeping it to mobile only.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Exclude or disable the patch that hides the fullscreen quick actions.

**Comment by Kobewaxman** [comment]
@oSumAtrIX I believe you may have misunderstood the patch request. They are asking to add quick actions for tablets, because there is no quick actions for tablets, even in stock YT.

**Comment by zzahkaboom24** [comment]
Yes, this is what I mean. Quick Actions in fullscreen does not seem to exist at all.
Here are the patches I applied, and as you can see, I havent applied any patch with the name "Hide fullscreen quick actions"

![1](https://github.com/revanced/revanced-patches/assets/42496614/aa89a0a3-fdb0-4bc8-b2cd-c1a1f17f1a83)
![2](https://github.com/revanced/revanced-patches/assets/42496614/07bc5546-3131-4a1b-9dcc-af6466c62eab)
